src/06_shared/ui/button/lightButton/lightButton.scss

```diff
@@ -0,0 +1,20 @@
+.light-button {
+  display: inline-block;
+  padding: 4px 8px;
+  font-size: var(--fs-base);
+  line-height: var(--lh-base);
+  color: rgb(var(--clr-pale-grey));
+  text-align: center;
+  background: transparent;
```

Comment:
Why background  does not have colour? Buttons aspirants should be the same on different place.

Reply:
What will be with a button if I add more text? Maybe it is better to add max-width style

Reply:
Button does not have background color in Figma
![image](https://github.com/user-attachments/assets/9439babb-a18d-49b5-916d-3d03607ad78c)
![image](https://github.com/user-attachments/assets/18bac23c-97b1-4d5e-86a7-649e35055edf)

Reply:
Ok, it is fine/ I was wrong